setup.py

```diff
@@ -1,12 +1,21 @@
+from os import path
 from setuptools import find_packages, setup
 
+# Get the long description from README.md
+PATH = path.abspath(path.dirname(__file__))
+with open(path.join(PATH, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
 setup(
     name="treedoc",
     version="0.2.0",
-    description="Minimalistic Python documentation in a tree structure.",
-    long_description="Minimalistic Python documentation in a tree structure.",
+    description="Minimalistic Python documentation for dendrophiles.",
+    long_description=long_description,
+    url="https://github.com/tommyod/treedoc",
     author="tommyod",
     author_email="tod001@uib.no",
+    maintainer="tommyod, smu095, glemvik",
+    maintainer_email="treedoc.dev@gmail.com",
     license="MIT",
```

Comment:
Let's decide on a license and upload a LICENSE file too. See [choosealicense.com](https://choosealicense.com/licenses/). MIT is very permissive, and I see no reason not to use it. Thoughts?

Reply:
No strong feelings, as far as I can tell the MIT license is quite popular. I suggest we go for it. I've made a `LICENCE.md`, will upload next commit.

Reply:
Great!